ontology/uco/deployment/deployment.ttl

```diff
@@ -0,0 +1,61 @@
+# imports: https://ontology.unifiedcyberontology.org/uco/observable/1.3.0
+
+@prefix core: <https://ontology.unifiedcyberontology.org/uco/core/> .
+@prefix deployment: <https://ontology.unifiedcyberontology.org/uco/deployment/> .
+@prefix observable: <https://ontology.unifiedcyberontology.org/uco/observable/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<https://ontology.unifiedcyberontology.org/uco/deployment>
+	a owl:Ontology ;
+	rdfs:label "uco-deployment"@en ;
+	rdfs:comment "TODO"@en ;
+	owl:imports observable:1.3.0 ;
```

Comment:
I set the Deployment ontology to import the Observable ontology because of the dependence in `deploymentAction`.

Why is deployment getting an entire namespace, instead of being in the Observable namespace?  Why is `Deployment` not an `ObservableObject` subclass?